lib/cropduster.js

```diff
@@ -29,6 +29,22 @@ window.CD = {
     });
   },
 
+  throwError: function(msg) {
+    if(typeof(MICapture) != "undefined") {
```

Comment:
I realize now that that would be `CD.error`, not `CD.log` and that `CD.error` does not currently exist.  The consistency question still stands though.